frontend/src/scenes/organization/TeamMembers/BulkInviteModal.tsx

```diff
@@ -53,67 +57,92 @@ function InviteRow({ index }: { index: number }): JSX.Element {
                     }}
                 />
             </Col>
+            <Col xs={1}>
+                <CloseOutlined
+                    style={isDeletable ? { color: red.primary } : { opacity: 0.5 }}
+                    onClick={isDeletable ? () => deleteInviteAtIndex(index) : undefined}
+                />
+            </Col>
         </Row>
     )
 }
 
 export function BulkInviteModal({ visible, onClose }: { visible: boolean; onClose: () => void }): JSX.Element {
     const { user } = useValues(userLogic)
     const { invites, canSubmit, invitedTeamMembersLoading, invitedTeamMembers } = useValues(bulkInviteLogic)
-    const { addMoreInvites, resetInvites, inviteTeamMembers } = useActions(bulkInviteLogic)
+    const { appendInviteRow, resetInviteRows, inviteTeamMembers } = useActions(bulkInviteLogic)
 
     useEffect(() => {
-        if (invitedTeamMembers.invites.length) {
+        if (invitedTeamMembers.length) {
             onClose()
         }
     }, [invitedTeamMembers])
 
     return (
-        <>
-            <Modal
-                title={`Invite your team members${user?.organization ? ' to ' + user?.organization?.name : ''}`}
-                visible={visible}
-                onCancel={() => {
-                    resetInvites()
-                    onClose()
-                }}
-                onOk={inviteTeamMembers}
-                okText="Invite team members"
-                destroyOnClose
-                okButtonProps={{ disabled: !canSubmit, loading: invitedTeamMembersLoading }}
-                cancelButtonProps={{ disabled: invitedTeamMembersLoading }}
-                closable={!invitedTeamMembersLoading}
-            >
-                <div className="bulk-invite-modal">
-                    <div>
-                        Invite as many team members as you want. <b>Names are optional</b>, but it will speed up the
-                        process for your teammates.
-                    </div>
-                    <Row gutter={16} className="mt">
-                        <Col xs={12}>
-                            <b>Email (required)</b>
-                        </Col>
-                        <Col xs={12}>
-                            <b>First Name</b>
-                        </Col>
-                    </Row>
+        <Modal
+            title={`Inviting team members${user?.organization ? ' to ' + user?.organization?.name : ''}`}
+            visible={visible}
+            onCancel={() => {
+                resetInviteRows()
+                onClose()
+            }}
+            onOk={inviteTeamMembers}
+            okText={`Invite ${pluralize(
+                invites.filter((invite) => invite.isValid && invite.target_email).length,
```

Comment:
This looks a bit strange when it's 0. I think we need to handle the zero-case specially.

Reply:
I thought it's a bit weird too. Ideas welcome, as I didn't have any good one! :(

Reply:
I just made it show "Invite team members" without number in this specific case.